lib/api/core/plugins/pluginsManager.js

```diff
@@ -831,8 +831,12 @@ class PluginsManager {
  * @returns {Promise}
  */
 function triggerHooks(emitter, event, data) {
+  const wildcardEvents = getWildcardEvents(event);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Move the declaration of "getWildcardEvents" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)